config/providers.yml

```diff
@@ -7,5 +7,12 @@
       {display_name: eu-north-h1, internal_name: hetzner-hel1, visible: true },
       {display_name: github-runners, internal_name: github-runners, visible: false }
     ]
+  },
+  {
+    provider_display_name: Leaseweb,
+    provider_internal_name: leaseweb,
+    locations: [
+      {display_name: us-central-l1, internal_name: leaseweb-wdc02, visible: true },
```

Comment:
Why isn't it `us-central-l2` instead of `us-central-l1`, given that it's `wdc02`?

Is Washington DC more central US than east US?"

Reply:
Yes, I mixed them :D the correct one "Is Washington DC more east US than central US?"

Reply:
Updated the names after checking with Umur.